### ISSUE TITLE
Using SI units

### ISSUE THREAD

**Issue by Hjorthmedh** [AllenInstitute/sonata#66]
Would it be possible to switch the SONATA format to SI units instead of natural units? I know a lot of simulators still use natural units, but there are so many potentials for mistakes.

**Comment by apdavison** [comment]
I am not in favour of this.

My feeling is that using unprefixed SI units (note that what you're calling natural units are also SI units; we're not talking feet and inches!) will introduce _more_ potential for mistakes, because (i) you end up doing more conversions and (ii) you introduce rounding errors due to floating point. 

However, we do need to ensure that the units are specified explicitly as metadata, as close to the data as possible.

**Comment by hernando** [comment]
I agree with Andrew. Units should not be implicit, but the there should be metadata that specifies the units. In the simulation output part we have included units metadata in all datasets. However, I think we are missing it in the simulation config json. 
Conversions may be unavoidable if different groups want to exchange data using different units though.
